docs/containers-certs.d.5.md

```diff
@@ -0,0 +1,34 @@
+% containers-certs.d(5)
+
+# NAME
+containers-certs.d - Directory for storing custom container-registry certficates
+
+# DESCRIPTION
+A custom certificate for a container registry can be configured by creating a directory under `/etc/containers/certs.d`.
+The name of the directory must correspond to the `host:port` of the registry (e.g., `my-registry.com:5000`).
+
+## Directory Structure
+A certs directory can contain one or more CA roots (i.e., `.crt` files) and one or more `<client>.{cert,key}` cert pairs.
+If more than one root or pair is specified, they will be used in alpha-numerical order.
```

Comment:
I don’t really think trusted CAs have an “order”; if the server provides enough certificates to build a path (handwaving) from the leaf to any of the trusted CAs, it is accepted.

For the cert+key, they are primarily filtered by the CAs the server sends as acceptable for client authentication, but then yes, the order seems to be by ASCII code comparison.